docs/conf.md

```diff
@@ -0,0 +1,116 @@
+
+**Docker**
+
+Additionally, the following optional envvars MUST be set/changed when running
+on Docker, since `localhost` is contained within the container:
+
+* `CACHE_DEFAULT`
+* `CACHE_AXES`
+* `EMAIL_HOST`
+
+### Optional
+
+* `SITE_ID`: defaults to `1`. The database ID of the site object. You usually
+  won't have to touch this.
+
+* `DEBUG`: defaults to `False`. Only set this to `True` on a local development
+  environment. Various other security settings are derived from this setting!
+
+* `IS_HTTPS`: defaults to the inverse of `DEBUG`. Used to construct absolute
+  URLs and controls a variety of security settings.
+
+* `DB_HOST`: hostname of the PostgreSQL database. Defaults to `localhost`,
+  unless you're using the `docker` environment, then it defaults to `db`.
+
+* `DB_USER`: username of the database user. Defaults to `openzaak`,
+  unless you're using the `docker` environment, then it defaults to `postgres`.
+
+* `DB_PASSWORD`: password of the database user. Defaults to `openzaak`,
+  unless you're using the `docker` environment, then it defaults to no password.
+
+* `DB_NAME`: name of the PostgreSQL database. Defaults to `openzaak`,
+  unless you're using the `docker` environment, then it defaults to `postgres`.
+
+* `DB_PORT`: port number of the database, defaults to `5432`.
+
+* `CACHE_DEFAULT`: redis cache address for the default cache. Defaults to
+  `localhost:6379/0`.
+
+* `CACHE_AXES`: redis cache address for the brute force login protection cache.
+  Defaults to `localhost:6379/0`.
+
+* `EMAIL_HOST`: hostname for the outgoing e-mail server. Defaults to
+  `localhost`.
+
+* `EMAIL_PORT`: port number of the outgoing e-mail server. Defaults to `25`.
+  Note that if you're on Google Cloud, sending e-mail via port 25 is completely
+  blocked and you should use 487 for TLS.
+
+* `EMAIL_HOST_USER`: username to connect to the mail server. Default empty.
+
+* `EMAIL_HOST_PASSWORD`: password to connect to the mail server. Default empty.
+
+* `EMAIL_USE_TLS`: whether to use TLS or not to connect to the mail server.
+  Defaults to `False`. Should be `True` if you're changing the `EMAIL_PORT` to
+  `487`.
+
+* `MIN_UPLOAD_SIZE`: the max allowed size of POST bodies, in bytes. Defaults to
+  4GB. Note that you should also configure your web server to allow this.
+
+* `SENDFILE_BACKEND`: which backend to use for authorization-secured upload
+  downloads. Defaults to `sendfile.backends.nginx`. See
+  (django-sendfile2)[https://pypi.org/project/django-sendfile2/] for available
+  backends.
+
+* `SENTRY_DSN`: URL of the sentry project to send error reports to. Default
+  empty, i.e. -> no monitoring set up. Highly recommended to configure this.
+
+## Specifying the environment variables
+
+There are two strategies to specify the environment variables:
+
+* provide them in a `.env` file
+* start the Open Zaak processes (with uwsgi/gunicorn/celery) in a process
+  manager that defines the environment variables
+
+### Providing a .env file
```

Comment:
Can you add `.env-example` as well to the branch? It should have all these constants imo.

Reply:
Sure thing